online-checkout/server/cs/server.cs

```diff
@@ -1,51 +1,18 @@
-namespace Server.Controllers;
+// The sandbox API key passed in 'ApiKeyClientAuthorization' is public.
+// Don't submit any personally identifiable information in any requests made with this key.
+// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+// for secure testing.
 
-[ApiController]
-[Route("[controller]")]
-public class CheckoutController : ControllerBase
-{
-    private readonly ILogger<CheckoutController> _logger;
-    private readonly IPaymentIntentsClient _paymentIntentsClient;
-
-    public CheckoutController(ILogger<CheckoutController> logger, IPaymentIntentsClient paymentIntentsClient)
-    {
-        _logger = logger ?? throw new ArgumentNullException(nameof(logger));
-        _paymentIntentsClient = paymentIntentsClient ?? throw new ArgumentNullException(nameof(paymentIntentsClient));
-    }
+using Dojo.Net;
 
-    [HttpPost]
-    public async Task<string> CheckoutAsync(CheckoutRequest checkoutRequest, CancellationToken cancellationToken)
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
+var result = await paymentIntentsClient.CreatePaymentIntentAsync(new CreatePaymentIntentRequest
+{
+    Amount = new()
     {
-        try
-        {
-            var result = await _paymentIntentsClient.CreatePaymentIntentAsync(new CreatePaymentIntentRequest
-            {
-                Amount = new Money()
-                {
-                    Value = checkoutRequest.Amount,
-                    CurrencyCode = "GBP"
-                },
-                Config = new PaymentIntentConfig()
-                {
-                    CancelUrl = new Uri(checkoutRequest.CancelUrl),
-                    RedirectUrl = new Uri(checkoutRequest.RedirectUrl),
-                },
-                Description = checkoutRequest.Description,
-                Reference = Guid.NewGuid().ToString() // can be you order id
-            }, cancellationToken);
-
-            return result.Id;
-        }
-        catch (ApiClientException<ProblemDetails> e)
-        {
-            // Check Dojo Documentation for error handling https://docs.dojo.tech/api#section/Errors
-            _logger.LogError(e, $"StatusCode:{e.Result.Status}, Response: {e.Result.Detail}, TraceId: {e.Result.TraceId}");
-            throw;
-        }
-        catch (ApiClientException e)
-        {
-            _logger.LogError(e, "Unhandled error");
-            throw;
-        }
-    }
-}
+        Value = 1000,
+        CurrencyCode = "GBP"
+    },
+    Reference = "Order 245",
+    CaptureMode = CaptureMode.Auto,
```

Comment:
This session is specific for auto/sale payments? If not, not sure if this field is needed.

For example in the virtual-terminal example you don't provide this since the `Auto` is the default value for it.

Reply:
We are only demonstrating for any case (Auto/Manual). We can remove the Auto mode. The same for Python we'd have to do. Should we add a comment or just remove Auto mode mention?

Reply:
'auto' need to be removed

Reply:
Done